nima/tests/integration/webserver/webserver/src/test/java/io/helidon/nima/tests/integration/server/ContentEncodingDisabledAbstract.java

```diff
@@ -0,0 +1,107 @@
+/*
+ * Copyright (c) 2023 Oracle and/or its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.helidon.nima.tests.integration.server;
+
+import java.util.NoSuchElementException;
+
+import io.helidon.common.http.Headers;
+import io.helidon.nima.http.encoding.ContentDecoder;
+import io.helidon.nima.http.encoding.ContentEncoder;
+import io.helidon.nima.http.encoding.ContentEncodingContext;
+import io.helidon.nima.testing.junit5.webserver.SetUpRoute;
+import io.helidon.nima.webclient.http1.Http1Client;
+import io.helidon.nima.webserver.http.HttpRules;
+import io.helidon.nima.webserver.http.ServerRequest;
+import io.helidon.nima.webserver.http.ServerResponse;
+
+/**
+ * Common code for tests to verify server response when
+ * <ul>
+ *     <li>content encoding is completely disabled,</li>
+ *     <li>request contains Content-Encoding header</li>
+ *     <li>and additional data with value of Content-Length header > 0</li>
+ * </ul>
+ */
+public abstract class ContentEncodingDisabledAbstract {
+
+    private final Http1Client client;
+
+    ContentEncodingDisabledAbstract(Http1Client client) {
+        this.client = client;
+    }
+
+    @SetUpRoute
+    static void routing(HttpRules rules) {
+        rules.any(ContentEncodingDisabledAbstract::handleRequest);
+    }
+
+    private static void handleRequest(ServerRequest request, ServerResponse response) {
+        response.send("response");
+    }
+
+    Http1Client client() {
+        return client;
+    }
+
+    static EmptyEncodingContext emptyEncodingContext() {
+        return new EmptyEncodingContext();
+    }
+
+    // Completely disable encoding. Even default "identity" encoding shall not be present.
```

Comment:
Using /** */ should be more consistent as class comment

Reply:
This is package private class so javadoc comment is not needed.It's just a note for anyone who will look into the code.